src/pmlogger/pmlogger_farm_check.timer

```diff
@@ -2,10 +2,10 @@
 Description=5 minute check of pmlogger farm instances
 
 [Timer]
-# if enabled, runs 1m after boot and every 5 mins
+# if enabled, runs 1m after boot and every 2 mins
 OnBootSec=1min
-OnCalendar=*:00/5
+OnCalendar=*:00/2
 
 [Install]
 WantedBy=timers.target
-RequiredBy=pmlogger_farm.service
+RequiredBy=pmlogger_farm.service pmlogger_farm_check.service
```

Comment:
Just to follow-up on our discussion, we think this isn't required/correct after all because there is an existing BindsTo=pmlogger_farm_check.timer in the pmlogger_farm_check.service

This is also introducing inconsistency with the way the primary pmlogger systemd state is setup, which feels incorrect and like the underlying problem is not understood (i.e. why doesn't it fail for the primary pmlogger without this...?).